docs/releases/v2020.1.rst

```diff
@@ -0,0 +1,193 @@
+
+mpc85xx-p1020
+~~~~~~~~~~~~~
+
+-  Enterasys WS-AP3710i
+-  OCEDO Panda
+
+ramips-mt7620
+~~~~~~~~~~~~~
+
+-  TP-Link Archer C2 (v1)
+-  TP-Link Archer C20 (v1)
+-  TP-Link Archer C20i
+-  TP-Link Archer C50 (v1)
+-  Xiaomi MiWifi Mini
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+-  Netgear EX6150 (v1)
+-  Netgear R6220
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+-  GL.iNet VIXMINI
+-  TP-Link TL-MR3020 (v3)
+-  TP-Link TL-WA801ND (v5)
+-  TP-Link TL-WR902AC (v3)
+
+Removed Hardware support
+------------------------
+
+-  ALFA Network Hornet-UB
+-  ALFA Network Tube2H
+-  ALFA Network N2
+-  ALFA Network N5
+
+Major changes
+-------------
+
+OpenWrt 19.07
+~~~~~~~~~~~~~
+
+Gluon v2020.1 is the first release to use OpenWrt 19.07. All targets
+therefore use Linux 4.14.166.
+
+batman-adv compat v14 removal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With 2020.1 we drop support for the long deprecated compat14 version of
+batman-adv. Communities still using this version should migrate to
+batman-adv- using the scheduled domain switch.
+
+IBSS wireless mesh removal
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Gluon 2020.1 drops support for the IBSS wireless protocol. Communities
+still using IBSS are suggested to migrate to 802.11s using the scheduled
+domain switch.
+
+Performance enhancements
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Gluon now installs zram-swap by default on ``ar71xx`` devices with 8MB
+of flash and 32M of RAM.
+
+Target renaming
+~~~~~~~~~~~~~~~
+
+- The ``ipq40xx`` target was renamed to ``ipq40xx-generic``.
+- The ``ipq806x`` target was renamed to ``ipq806x-generic``.
+
+Status Page
+~~~~~~~~~~~
+
+The status page now includes information about the nexthop prettyname,
+mac address and interface towards the selected gateway when batman-adv
+is used. It also shows the site and additionally the domain name if the
+node is in a multidomain setup.
+
+DECT button to enter config mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many AVM devices don’t feature a separate RESET/WPS button, so starting
+this release we support entering the config mode via DECT buttons.
+
+Bugfixes
+--------
+
+Autoupdater aliases
+~~~~~~~~~~~~~~~~~~~
+
+We have added several new aliases for continued autoupdater
+compatibility on the following devices:
+
+- Ubiquiti UniFi AC LR
+- Raspberry Pi
+
+Site changes
+------------
+
+site.mk
+~~~~~~~
+
+-  The ``GLUON_WLAN_MESH`` variable can be dropped, as 802.11s is now
+   the only supported wireless transport.
+
+Internals
+---------
+
+x86 partition size
+~~~~~~~~~~~~~~~~~~
+
+The x86 partition size has been reduced to fit on disk with 128 MB
+capacity.
+
+Linting Targets
+~~~~~~~~~~~~~~~
+
+Support for linter make targets was added.
+
+- ``make lint``
+- ``make lint-sh`` to only check shell scripts
+- ``make lint-lua`` to only check lua scripts
+
+These require the shellcheck and luacheck tools. The docker image has
+been updated accordingly.
+
+Continuous integration
+~~~~~~~~~~~~~~~~~~~~~~
+
+We now utilize continuous integration testing using Jenkins and thereby
```

Comment:
We utilize continuous integration testing using Jenkins now

(Either "time" at the beginning of the sentence if the sentence is long, but preferably "time" should be stated at the end of a sentence)

Reply:
Do you have a reference for that rule?

Reply:
E.g. in your sentence you use "now" as an adverb, but it's usually an adjective. Your sentence is correct, but sounds strange.
https://wordtype.org/of/now

Reply:
I rebuilt the sentence.